eng/Versions.props

```diff
@@ -12,127 +12,77 @@
     <VersionPrefix>2.0.0</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <!--ML.NET Core dependencies-->
-    <NewtonsoftJsonPackageVersion>10.0.3</NewtonsoftJsonPackageVersion>
-    <SystemCodeDomPackageVersion>4.5.0</SystemCodeDomPackageVersion>
-    <SystemReflectionEmitLightweightPackageVersion>4.3.0</SystemReflectionEmitLightweightPackageVersion>
-    <SystemThreadingChannelsPackageVersion>4.7.1</SystemThreadingChannelsPackageVersion>
-    <!-- Other/External dependencies -->
-    <GoogleProtobufPackageVersion>3.19.4</GoogleProtobufPackageVersion>
-    <LightGBMPackageVersion>2.3.1</LightGBMPackageVersion>
-    <MicrosoftExtensionsPackageVersion>2.1.0</MicrosoftExtensionsPackageVersion>
-    <MicrosoftMLOnnxRuntimePackageVersion>1.10.0</MicrosoftMLOnnxRuntimePackageVersion>
-    <MlNetMklDepsPackageVersion>0.0.0.12</MlNetMklDepsPackageVersion>
-    <ParquetDotNetPackageVersion>2.1.3</ParquetDotNetPackageVersion>
-    <SystemDrawingCommonPackageVersion>4.5.0</SystemDrawingCommonPackageVersion>
+
```

Comment:
More generally, have we considered switching to the official NuGet support for managing versions?

https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-packages

Reply:
The current patterns are established by Arcade.  We could see about other patterns to follow moving forward, but right now we're taking some incremental steps.

Reply:
Logged https://github.com/dotnet/arcade/issues/9153

Reply:
Thanks @tannergooding